app/shared-components/ForgotPassword/ForgotPassword.scss

```diff
@@ -0,0 +1,169 @@
+/* FORGOT PASSWORD PANEL */
```

Comment:
@benstrumeyer add a document header to this file please

Reply:
Added